package proto_ff;

import "yd_fieldoptions.proto";


message serverHornserverHorn
{
	optional int64 id = 1[(yd_fieldoptions.field_cname) = "喇叭ID"];
	optional int32 time = 2[(yd_fieldoptions.field_cname) = "持续时间S"];
	optional int32 minTime = 3[(yd_fieldoptions.field_cname) = "最短持续时间S"];
	optional int64 itemid = 4[(yd_fieldoptions.field_cname) = "消耗道具"];
	optional int32 currencyType = 5[(yd_fieldoptions.field_cname) = "替代货币类型"];
	optional int32 currencyCost = 6[(yd_fieldoptions.field_cname) = "替代货币价格"];
}


message Sheet_serverHornserverHorn
{
	repeated serverHornserverHorn serverHornserverHorn_List = 1[(yd_fieldoptions.field_arysize)=107];
}
